tests/conftest.py

```
@@ -406,15 +493,38 @@ def __init__(self) -> None:
         """Avoid requiring a full Home Assistant instance for registry tests."""
 
     def async_get(self, entity_id_or_uuid: str) -> None:
-        """Return no registry entry for isolated unit tests."""
+        """Return no registry entry for isolated unit tests.
+
+        Args:
+            entity_id_or_uuid (str):
+                Entity ID or registry UUID to resolve.
+        """
 
     def async_get_entity_id(self, domain: str, platform: str, unique_id: str) -> None:
```

Ruff (0.16.1)

[warning] Unused method argument: `domain`

(ARG002)